File: Contains Duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        int l = nums.length;
        boolean op = false;
        HashMap<Integer, Integer> newMap = new HashMap<>();

        for(int i = 0; i < l; i++){
            int value = nums[i];
                if(newMap.containsKey(value)){
                    op = true;
                }else{
                    newMap.put(nums[i], i);
                                        //return false;
                }

        } 
        return op;
    }
}

File: ContainsDuplicate.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        map = set()
        for num in nums:
            if num in map:
                return True
            map.add(num)
        return False
        
